instrumentation/grpc/Gemfile

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+source 'https://rubygems.org'
+
+gemspec
+
+gem 'opentelemetry-api', path: '../../api'
+gem 'opentelemetry-instrumentation-base', path: '../base'
+
+group :test do
+  gem 'opentelemetry-common', path: '../../common'
+  gem 'opentelemetry-sdk', path: '../../sdk'
+  gem 'opentelemetry-semantic_conventions', path: '../../semantic_conventions'
+
+  gem 'pry'
```

Comment:
I think we are moving these to the Gemspec as development dependencies. E.g.
https://github.com/open-telemetry/opentelemetry-ruby/pull/896/files#diff-0e18c3a8e005f4da1b58da58cdd9c29ef449fb3526504bcb81d70893de6e078aR36

Reply:
Oh, good point. Looks like it was already in the gemspec with a matching line in the `test_helper.rb`, so I just removed this in 8aaea741 .